WinImageBuilder.psm1

```diff
@@ -38,15 +38,16 @@ $VirtIODrivers = @("balloon", "netkvm", "pvpanic", "qemupciserial", "qxl",
              "qxldod", "vioinput", "viorng", "vioscsi", "vioserial", "viostor")
 
 $VirtIODriverMappings = @{
-    "2k8" = @(60, 1);
-    "2k8r2" = @(61, 1);
-    "w7" = @(61, 0);
-    "2k12" = @(62, 1);
-    "w8" = @(62, 0);
-    "2k12r2" = @(63, 1);
-    "w8.1" = @(63, 0);
-    "2k16" = @(100, 1);
-    "w10" = @(100, 0);
+    "2k8" = @(60, 0, 1);
+    "2k8r2" = @(61, 0, 1);
+    "w7" = @(61, 0, 0);
+    "2k12" = @(62, 0, 1);
+    "w8" = @(62, 0, 0);
+    "2k12r2" = @(63, 0, 1);
+    "w8.1" = @(63, 0, 0);
+    "2k16" = @(100, 14393, 1);
+    "2k19" = @(100, 17763, 1);
+    "w10" = @(100, 0, 0);

```

Comment:
there is no need to rewrite all these arrays, just add 2k19 key with 3 values and modify 2k16 with an additional value.

please try to keep backward compatibility in method params or arrays.